spec/models/user_spec.rb

```diff
@@ -59,6 +59,7 @@
       create(:donor, admin: true, name: "Admin/Donor")
       create(:admin, name: "Admin")
       create(:cyclist, name: "Cyclist")
+      create(:donation, donor: donor)
```

Comment:
I'm confused as to why this change to the test covers the corresponding change to the model.